vmdb/app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -298,4 +299,33 @@ def save_storage_files_inventory(storage, hashes)
     deletes = storage.storage_files(true).dup
     save_inventory_multi(:storage_files, storage, hashes, deletes, :name)
   end
+
+  def save_orchestration_stacks_inventory(ems, hashes, target = nil)
```

Comment:
The only difference I can see between this save_orchestration_stacks_inventory and the same method in save_inventory_cloud is that the cloud inventory stacks have a template_id.

Is there a way to reuse that method? Maybe move orchestration to save_inventory_helper or just save_inventory (like save_vms_inventory).

Also, do Openstack Infra Stacks not have a template? Is that specific to cloud orchestration stacks?